Code/Engine/Core/Scripting/ScriptClassResource.h

```diff
@@ -0,0 +1,60 @@
+#pragma once
+
+#include <Core/ResourceManager/Resource.h>
+#include <Foundation/Containers/ArrayMap.h>
+
+class ezWorld;
+using ezScriptClassResourceHandle = ezTypedResourceHandle<class ezScriptClassResource>;
+
+class EZ_CORE_DLL ezScriptInstance
```

Comment:
documentation comments please

Reply:
This classes are still very work in progress. I will add comments once they are more finalized.